app/models/service_ansible_tower.rb

```diff
@@ -1,4 +1,5 @@
 class ServiceAnsibleTower < Service
+  include ApiAnsibleMixin
```

Comment:
@lfu should this file name indicate that its ExtraVars maybe ApiExtraVars. Since most of the fields in there are going in as ExtraVars for connecting to our REST API.

Reply:
I'm not good at naming. But I think ```ApiAnsibleMixin``` makes more sense than ```ApiExtraVarsMixin``` in terms of audience. People may have heard of ```ansible``` but not ```extra_vars```. 

@gmcculloug Ideas?

Reply:
Renamed the module to ```AnsibleExtraVarsMixin```.